profiles/full.props

```diff
@@ -1,23 +1,23 @@
 <Project>
   <!-- Everything -->
   <PropertyGroup Condition="'$(OS)' == 'Windows_NT'">
-    <FakeItEasyTargetFrameworks>net462;netstandard2.0;netstandard2.1;net6.0</FakeItEasyTargetFrameworks>
+    <FakeItEasyTargetFrameworks>net462;netstandard2.0;netstandard2.1;net6.0;net8.0</FakeItEasyTargetFrameworks>
     <ValueTaskExtensionsTargetFrameworks>net462;netstandard2.0;netstandard2.1;net6.0</ValueTaskExtensionsTargetFrameworks>

```

Comment:
I'm tempted to experiment with dropping net6.0 and netstandard2.1. Not sure this is the time though

Reply:
Probably not...